Dockerfile

```diff
@@ -2,6 +2,12 @@ FROM alpine:3.6
 
 WORKDIR /opt/issue-sync
 
+RUN apk update --no-cache && apk add ca-certificates
+
 COPY bin/issue-sync /opt/issue-sync/issue-sync
 
-ENTRYPOINT ./issue-sync
+COPY config.json /opt/issue-sync/is-config.json
```

Comment:
just curious, if the config is named `config.json` on the host, why mount it as `is-config.json` in the container? it seems cleaner to drop the prefix everywhere.